Dixit_Logic/Classes/Card.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dixit_Logic.Interfaces;
+using System.Drawing;
+
+namespace Dixit_Logic.Classes
+{
+    /// <summary>
+    /// This class a structure of a gamecard. A Card object will store an image and an unique
+    /// indentifier. This values only assigned by the constructor. The image and the identifier
+    /// cannot be changed after an object is constructed.
+    /// </summary>
+    public class Card : ICard
+    {
+        /// <summary>
+        ///  The card indentifier
+        /// </summary>
+        private int _id;
+
+        /// <summary>
+        /// The image of the card
+        /// </summary>
+        Bitmap _image;
```

Comment:
There is no use having this field anymore, Bitmap cannot be used as source, we need an IValueConverter for that, but I already implemented one that can make ImageSource from an ID in https://github.com/VSZM/Dixit_By_MonkeyzWithKeyboardz_ELTE/pull/42

It is unnecessary to contain the images in Dixit_Logic when id is enough